### DISCUSSION TITLE
[Enhancement] [input-number] Add readonly for el-input-number

### DISCUSSION THREAD

**Discussion by jarhan-pro** [element-plus/element-plus#9534] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `input-number`

## Description

Is it possible to add the readonly attribute on the el-input-number component like you have it on the el-input?

### Additional comments

_(empty)_

**Comment by holazz** [comment]
You can use `disabled`.

**Reply by jarhan-pro** [reply]
Yes, but disabled is not the same as readonly. With disabled you can't focus on the input and this is what I need

**Reply by holazz** [reply]
`readonly` will be available in the next release, related https://github.com/element-plus/element-plus/pull/9545

**Reply by jarhan-pro** [reply]
Great! Thanks!
